python/check_address.py

```diff
@@ -52,14 +53,42 @@ def handle_msg(self, msg_pmt):
         else:
             address = packet[7:14]
 
+        if len(packet) > 20:
+
+            hbit = packet[20] >> 7
+# digi callsign; and above: hbit  (most significant bit after digi callsign)
+# (set to 1 in case of digipeated message)
+            digi = [c >> 1 for c in packet[14:20]]
+
+# error handling
+            try:
+                digi = bytes(digi).decode('ascii').rstrip(' ')
+
+            except UnicodeDecodeError:
+                print("Not to ASCII convertable string detected.")
```

Comment:
How can this possibly fail to decode as ASCII? It is a `bytes` object where all the bytes have the MSB equal to zero. Regardless, in the event there was a `UnicodeDecodeError` exception, `digi` would be undefined but potentially read below.

Reply:
You are right. Then we just remove it.